### PR TITLE
Add order history product image and link to pdp

### PR DESCRIPTION
**Pull request by lortschi**
This belongs to my order history product image and back to pdp extension.

### CODE REVIEW

**Review by vamcart — APPROVED** [review]

File: config/server.js [hunk 1]
```diff
@@ -30,13 +30,13 @@ module.exports = {
 	mongodbServerUrl: dbUrl,
 
 	smtpServer: {
-		host: '',
-		port: 0,
+		host: 'smtp.gmail.com',
```
Comment (ngoyal16) [inline config/server.js]: do not add your personal creds to the system

**Review by ngoyal16 — CHANGES_REQUESTED** [review]

**Comment by ngoyal16** [comment]
add server.js again to repo

File: package.json [hunk 2]
```diff
@@ -44,7 +44,11 @@
     "lint:fix": "eslint . --fix",
     "prettier": "prettier --write \"src/**/*.js\" \"theme/**/*.js\"",
     "precommit": "lint-staged",
-    "start": "node -r esm src/index.js"
+    "start": "node -r esm src/index.js",
```
Comment (ngoyal16) [inline package.json]: remove unnecessary script form package.json.

File: package.json [hunk 3]
```diff
@@ -88,6 +92,7 @@
     "lodash": "^4.17.10",
     "lru-cache": "^4.1.3",
     "lscache": "^1.3.0",
+    "material-ui": "^0.20.2",
```
Comment (ngoyal16) [inline package.json]: material-ui is not a part of api

**Review by ngoyal16 — CHANGES_REQUESTED** [review]

**Review by vamcart — APPROVED** [review]

**Pull request merged**